components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIGatewayManager.java

```diff
@@ -119,6 +119,30 @@ private void sendUnDeploymentEvent(APIProduct apiProduct, String tenantDomain, S
         APIUtil.sendNotification(deployAPIInGatewayEvent, APIConstants.NotifierType.GATEWAY_PUBLISHED_API.name());
     }
 
+    private void sendGatewayPolicyDeploymentEvent(String mappingUuid, String tenantDomain, Set<String> publishedGateways) {
```

Comment:
method level description missing. add to all the missing methods in the PR

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/12201/commits/1136a795360198388c6b4bd97029c20b0f898832

Thanks and Regards